Gemfile.lock

```diff
@@ -551,14 +551,14 @@ GEM
       actionpack (>= 3.1)
       railties (>= 3.1)
       slim (~> 3.0)
-    slim_lint (0.11.0)
+    slim_lint (0.12.0)
       rake (>= 10, < 13)
-      rubocop (>= 0.47.0)
+      rubocop (>= 0.48.0)
       slim (~> 3.0)
       sysexits (~> 1.1)
     slop (3.6.0)
     socksify (1.7.1)
-    split (2.2.0)
+    split (3.0.0)
```

Comment:
This is a major version change. Have you run the app locally to see if the way we are using split still works? I believe the only place we are using split is on the SP landing page, which means you'll need to run the sp-rails app locally, then visit the IdP from the SP to see the landing page.